package.json

```diff
@@ -32,11 +32,11 @@
     "@radix-ui/react-separator": "0.1.0",
     "@skypack/package-check": "0.2.2",
     "@types/node": "16.9.6",
-    "@types/react": "17.0.33",
```

Comment:
Is this change required? i'd prefer not to update anything to react 18 as this might signal that the library is fully compatible with it

Reply:
I don’t recall. I’ll check today.

Reply:
@jgoz Thank you so much!

Reply:
Reverted. Thanks for calling this out. I forget why I did this originally &mdash; maybe I thought the error was related to react's types in some way. But it wasn't 😄